README.md

```diff
@@ -12,6 +12,67 @@ Here are some links to know more:
 
 The APIs are documented in [API.md](./API.md).
 
+## Develop and running the server locally
```

Comment:
I wondered if that should be in eg `RUNNING.md` that would be linked from the README, instead of embedding everything in the README. Tell me what you think!

Reply:
This is fine.